rotkehlchen/tests/exchanges/test_woo.py

```diff
@@ -56,7 +56,7 @@ def test_woo_assets_are_known(mock_woo):
 
 def test_query_online_trade_history_basic(mock_woo):
     """Assert that the expected arguments are passed to the `_api_query` method"""
-    start_ts, end_ts = (1634600000, 1634620000)
+    start_ts, end_ts = 1634600000, 1634620000
```

Comment:
this is not related to this change. State that in the description of the PR. Also why the test was passing before? That might indicate that something is wrong in the test logic?

Reply:
it was a nitpick from a previous pr, both are valid